salt/utils/thin.py

```diff
@@ -72,11 +69,13 @@
     except ImportError:
         ssl_match_hostname = None
 # pylint: enable=import-error,no-name-in-module
-
-
 if _six.PY2:
     import concurrent
+
```

Comment:
Do we still need this `if _six.PY2` check?

Reply:
Yes. This keeps support for https://docs.saltstack.com/en/master/topics/ssh/ssh_ext_alternatives.html  which allows you to bundle up an old version of salt that uses python 2 to use on the target host. This PR removes salt-ssh trying to automatically add python2 libraries automatically, but if a user is intentionally using `ssh_ext_alternatives` I think they should still be able to add python 2 libraries for older versions of salt.